contracts/owner/Operator.sol

```diff
@@ -0,0 +1,77 @@
+pragma solidity ^0.6.2;
+
+import '@openzeppelin/contracts/GSN/Context.sol';
+import '@openzeppelin/contracts/access/Ownable.sol';
+
+contract Operator is Context, Ownable {
```

Comment:
It could be replaced with the `IAccessControl` in OZ repo. As for the specific roles, suggest to define in your own project.